calico-enterprise/reference/installation/tigerastatus.mdx

```diff
@@ -0,0 +1,62 @@
+---
+description: Descriptions of Tigera Status fields
+---
+
+# TigeraStatus
+
+Installing {{prodname}} on your Kubernetes cluster is managed by the Tigera operator. The operator get its configuration from the API resources and communicates through the Kubernetes API. The operator is deployed as a Deployment in the `tigera-operator` namespace, and records status in the `tigerastatus` resource. Depending on how your cluster is configured, you can get `tigerastatus` on these resources:
+
+- apiserver
+- applicationlayer
+- authentication
+- calico
+- calico-windows
+- compliance
+- egressgateway
+- intrusion-detection
+- log-collector
+- log-storage
+- manager
+- management-cluster-connection
+- policy-recommendation
+- secrets
+
+## Get TigeraStatus
+
+To get tigerastatus for a resource, run the following command:
+
+   `kubectl get tigerastatus`
+
+For detailed output (including messages and further details on any non-functioning components), run either of the following commands:
+
+   `kubectl describe tigerastatus X`
+
+   `kubectl get tigerastatus X -o yaml`
+
+## Log storage
+
+Log storage provides persistent storage for {{prodname}} Elasticsearch logs (flow, dns, l7, bpg, audit, etc.), and compliance reports.
+
+To check log storage status, run the following command:
+
+   `kubectl get logstorage.operator.tigera.io tigera-secure`
```

Comment:
I still think this is wrong, and should be:

```
kubectl get tigerastatus log-storage
```

Reply:
Fixed.